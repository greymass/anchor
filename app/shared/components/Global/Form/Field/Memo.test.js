import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import GlobalFormFieldMemo from './Memo';

const mockProps = {
  label: 'testing account field',
  loading: false,
  name: 'account_field',
};

describe('GlobalFormFieldMemo', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<GlobalFormFieldMemo {...mockProps} />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
