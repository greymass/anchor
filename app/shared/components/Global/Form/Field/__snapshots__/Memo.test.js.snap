// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GlobalFormFieldMemo renders correctly 1`] = `
<FormField
  control={[Function]}
  fluid={true}
  label="testing account field"
  loading={false}
  name="account_field"
  onChange={[Function]}
/>
`;
